src/main/java/com/thealgorithms/ciphers/Vigenere.java

```diff
@@ -6,66 +6,63 @@
  * @author straiffix
  * @author beingmartinbmc
  */
-public class Vigenere {
```

Comment:
Why did you split it? IMO, encoding and decoding have high cohesion and belong to the same class